### ISSUE TITLE
A stub can only resolve a value once.

### ISSUE THREAD

**Issue by astephenb** [substantial/sinon-stub-promise#4]
```javascript
stub = sinon.stub.returnsPromise().resolves('some thing');

someFunction(); // stub will resolve with 'some thing'
someFunction(); // stub will resolve with undefined
```

**Comment by amay** [comment]
What is someFunction in this example?

**Comment by astephenb** [comment]
`someFunction` is a promise under test that uses the `stub`.

**Comment by EmergentBehavior** [comment]
Had same issue for a sandbox stub.

**Comment by rhagigi** [comment]
Is this completed fixed by #12? If it is, it should be closed.
